### PR TITLE
Improve French translation

### PR DESCRIPTION
**Pull request by Otiel**
A few fixes, improvements and missing translations to the french localization strings.

### CODE REVIEW

**Comment by ajayyy** [comment]
Thank you! I will review it in a bit.

File: _locales/fr/messages.json [hunk 1]
```diff
@@ -109,21 +109,158 @@
...
+    },
+    "noticeUpdate2": {
+        "message": "Si elle ne vous plaît pas, cliquez sur le bouton \"Ne plus montrer\"",
+        "description": "The second line of the message displayed after the notice was upgraded."
+    },
+    "setStartSponsorShortcut": {
+        "message": "Changer le raccourci pour enregistrer un segment"
+    },
+    "setSubmitKeybind": {
+        "message": "Changer le raccourci pour soumettre les segments"
```
Comment (ajayyy) [inline _locales/fr/messages.json]: Is `raccourci` the best word to use here? Maybe `clé`?

File: _locales/fr/messages.json [hunk 2]
```diff
@@ -109,21 +109,158 @@
...
+    },
+    "keybindDescription": {
+        "message": "Appuyez sur une touche"
```
Comment (ajayyy) [inline _locales/fr/messages.json]: Would `clé` be better here?

File: _locales/fr/messages.json (hunk 1 above)
Comment (Otiel) [reply _locales/fr/messages.json]: No `key` is a false friend in French.

* Keyboard = clavier
* Key (from keyboard) = touche
* _Key (to open a door) = clé_
* Keyboard shortcut(s) = touche(s) de raccourci / raccourci(s) clavier

And like in English you often hear "shortcut" only (not keyboard shortcut), we usually say only "raccourci" when the context is enough to understand that we're talking about the keyboard.

File: _locales/fr/messages.json (hunk 2 above)
Comment (Otiel) [reply _locales/fr/messages.json]: See [my other comment](https://github.com/ajayyy/SponsorBlock/pull/164#discussion_r342191375): `raccourci` is the right term here.

File: _locales/fr/messages.json (hunk 1 above)
Comment (ajayyy) [reply _locales/fr/messages.json]: Okay, thanks for the explanation. I had not heard that term before.

**Pull request merged**